projects/app-lob/src/app/app.component.ts

```diff
@@ -13,9 +17,23 @@ export class AppComponent implements OnInit {
     private typefacesLoaded = ["Titillium Web", "Roboto"];
     private typefaceUrl = "https://fonts.googleapis.com/css?family=";
 
-    constructor(@Inject(DOCUMENT) private document: Document) {}
+    constructor(@Inject(DOCUMENT) private document: Document,  private router: Router, private activatedRoute: ActivatedRoute, private seoService: SEOService) {}
 
     public ngOnInit() {
+        this.router.events.pipe(
```

Comment:
This pipe chain of rxjs operators could be shorten into two operator invokes, like only filter and mergeMap